products/apache-camel.md

```diff
@@ -0,0 +1,218 @@
+---
+title: Apache Camel
+permalink: /apache-camel
+alternate_urls:
+-   /camel
+-   /camel-lang
+category: lang
+releasePolicyLink: https://camel.apache.org/blog/2020/03/LTS-Release-Schedule/
+changelogTemplate: https://camel.apache.org/releases/release-__LATEST__/
+releaseColumn: true
+releaseDateColumn: true
+eolColumn: Bug and Security Fixes
+
+auto:
+-   maven: org.apache.camel/camel
+
+# for LTS : eol = releaseDate + 1y
+# for non-LTS : eol = releaseDate (Camel never release patches for those)
```

Comment:
I am not sure if this is correct, i mean eol means the product suggest to migrate newly
supported version, if a version released its eol shouldnt be same date, but if exist next version
For example eol for 3.19 should be the release date of 3.20 but support might be ended but it is not eoled

Reply:
I am not sure either. Camel does not have a notion of active support / security support and I have read the following on https://camel.apache.org/blog/2020/03/LTS-Release-Schedule/ :

> For non-LTS releases then we plan to not release patch releases, for example there will not be 3.1.1, 3.1.2 patch releases for the non-LTS release of Camel 3.1.0.

This is the same as saying that the version is EOL. But I am not against your solution either.

@captn3m0 do you know if there is a similar product on endoflife.date ? And if there is what was decided for this product ?

Reply:
Ansible did something similar with a "EOL-on-arrival-patch-release" (4.10): https://github.com/ansible/ansible/issues/75884#issuecomment-988915630

Since only the LTS releases are considered production, it might be better to only list those. The non-LTS releases are essentially trunk releases, so we don't have to list them.

If we want to keep them, I'd prefer what we're listing currently (release date = eol date), just because we're certain that there will be no security fixes on these.

Reply:
i still insist to use eol for a product to say costumers not to use related product version and  for time to them to shift a newer still alive version.

So when we say eol = release date and that version is latest cycle ( which doesnt have any other cycle ) means dont use this product. 
Instead of this we should use this product cycle is still active ( not eol ) until next cycle released (with next cycle release date )
But it is not (because it is not ltf) supported { wont get any point release )

Reply:
I would prefer to keep the intermediate versions. But after a second thought I agree with @usta, it looks odd to mark the latest release EOL directly, so maybe we should mark the latest one supported until a new minor is released.

They said they plan to not release patch for non-LTS releases. But maybe we can still consider they would release one for the latest release if a security issue occurs (because there is no other alternative) ?

Reply:
but you are missing the main point here brother , support ( critical or normal ) and eol are different things. When a product tags a version as eol it means that there should be a newer version existed in it sametime so that suggest current users of eoled version will pass that one. That was my motivation that we should point eol dates as new cycle's release dates. 
A product might not have any point release patches or wont take a patch but still can be active ( Not EOLed )

Reply:
> A product might not have any point release patches or wont take a patch but still can be active ( Not EOLed )

I agree, I also updated the EOL dates for non-LTS. @usta, thanks for your insight, I let you do the (hopefully ! ;)) final review.